lighthouse-core/report/html/report-styles.css

```diff
@@ -263,6 +263,11 @@
   overflow-y: scroll;
   height: calc(100% - var(--topbar-height));
 }
+@media print {
+  .lh-devtools .lh-container {
+    height: inherit;
```

Comment:
i tried this out and i'm not sure it fixes the issue.
rather than height, i'm pretty sure we need to unset the `overflow` property back to 'visible' instead.

```css
@media print {
  .lh-devtools .lh-container {
    overflow-y: unset;
  }
}
```

Reply:
Try it out in this manner:

comment out the "printWindow.close()" fn, cancel the print prompt when it appears, and inspect the print window / apply these styles

Reply:
it was only barely working :)

fixed